src/clients/nfs_server.rb

```diff
@@ -303,7 +311,10 @@ def NfsServerDeleteHandler(options)
         end
       end
 
-      CommandLine.Print(_("Mount point not found.")) if !deleted
+      if !deleted
+        # TRANSLATORS %s is directory where it should be mounted, but it is mounted there.
```

Comment:
Um, I do not get it. Should it be "but it is not found" or something like that?

Reply:
well, I do not want to confuse them that directory is not found. In fact mount point is not found in nfs server configuration.